docker-compose.yml

```diff
@@ -13,29 +15,51 @@ services:
       - app
     volumes_from:
       - app
-    environment:
-      - VIRTUAL_HOST
+    env_file:
+      - frontend.env
     networks:
-    - back
-    - lb_web
+    - backend
+    - frontend-web
+    logging:
+      driver: none
   app:
     image: nextcloud:12-fpm
     links:
       - db
     volumes:
       - ./data/apps:/var/www/html/apps
       - ./data/config:/var/www/html/config
+      - ./data/custom_apps:/var/www/html/custom_apps
       - ./data/data:/var/www/html/data
     networks:
-    - back
+    - backend
+    logging:
+      driver: none
+  onlyoffice-document-server:
```

Comment:
We use collabora.
Do you like it better onlyoffice, still didn't understand the difference between the 2.
Actually for people, we try to make them use pads :)
Office is so 2000 :)